Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -1043,6 +1043,52 @@ void WindowManager::process_mouse_event(MouseEvent& event, Window*& hovered_wind
         clear_resize_candidate();
 }
 
+void WindowManager::reevaluate_hovered_window(Window* updated_window)
```

Comment:
Can we come up with a better name for this function?
Re-evaluate what?

Reply:
This function re-evaluate the hovered window, and changes it to another one if needed. I'm open to suggestions ;-) Can't think of a good name.

Reply:
Ohh I get it. I was reading it in a weird way. Yes, we are re-evaluating which window is hovered. Makes sense :)